jib-core/src/main/java/com/google/cloud/tools/jib/tar/TarExtractor.java

```diff
@@ -41,21 +44,20 @@
    */
   public static void extract(Path source, Path destination) throws IOException {
     String canonicalDestination = destination.toFile().getCanonicalPath();
-
     try (InputStream in = new BufferedInputStream(Files.newInputStream(source));
         TarArchiveInputStream tarArchiveInputStream = new TarArchiveInputStream(in)) {
-
+      Map<TarArchiveEntry, FileTime> modificationTimeMap = new HashMap();
```

Comment:
This gives me a warning.

```suggestion
      Map<TarArchiveEntry, FileTime> modificationTimeMap = new HashMap<>();
```